packages/common/src/ui/components/domain/StatusCrumbs/StatusCrumbs.stories.tsx

```diff
@@ -1,36 +1,30 @@
 import React, { useState } from 'react';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
-import { StatusCrumbs } from './StatusCrumbs';
-import { OutboundShipmentStatus } from '../../../..';
-import { LocaleKey, useTranslation } from '../../../../intl';
 import Stack from '@mui/material/Stack';
 import Radio from '@mui/material/Radio';
 import RadioGroup from '@mui/material/RadioGroup';
 import FormControlLabel from '@mui/material/FormControlLabel';
 import FormControl from '@mui/material/FormControl';
 import FormLabel from '@mui/material/FormLabel';
+import { StatusCrumbs } from './StatusCrumbs';
+import { LocaleKey, useTranslation } from '../../../../intl';
 
-const StatusTranslation: Record<OutboundShipmentStatus, LocaleKey> = {
+const statusTranslation: Record<string, LocaleKey> = {
```

Comment:
did some status changes a little further down so just changed this story up a bit..